### PR TITLE
removing problematic link and call to action

### PR DESCRIPTION
**Pull request by ToddKopriva**
I removed a call to action that was misplaced, and I did some copy-editing since I had the file open.

### CODE REVIEW

**Comment by ToddKopriva** [comment]
fixes issue #830

File: tutorials/cloudbuild-test-runner.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
 ---
-title: Using Google Cloud Build as a Test Runner
+title: Using Cloud Build as a test runner
 description: Learn how to use Google Cloud Build as a test runner.
 author: tmatsuo
 tags: Google Cloud Build, Testing, PHP
```
Comment (jennyrosewood) [inline tutorials/cloudbuild-test-runner.md]: Should we update the description and the tags while we're at it?

File: tutorials/cloudbuild-test-runner.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-For example, here is a simple command to build a docker image:
+For example, here is a simple command to build a Docker image:
 
     gcloud builds submit -t gcr.io/my-project/my-image .
 
-This command will send the files in the current directory to Google Cloud
+This command sends the files in the current directory to Google Cloud
 Storage, then on one of the Cloud Build VMs, fetch the source code, run
-`Docker build` and upload the image to [Google Container Registry][registry].
```
Comment (jennyrosewood) [inline tutorials/cloudbuild-test-runner.md]: Cloud Storage

File: tutorials/cloudbuild-test-runner.md [hunk 3]
```diff
@@ -31,22 +32,21 @@ described below.
...
 [config]: https://cloud.google.com/cloud-build/docs/config
 
 ## Running tests in a build step
 
-If we can use any arbitrary Docker image as the Build Step, and the source code
-is available, then we can run unit tests as a Build Step. By doing so, you always
-run the test with the same docker image. You don't have to worry about environment
-differences on CI systems any more.
+If we can use any arbitrary Docker image as the build step, and the source code
+is available, then we can run unit tests as a build step. By doing so, you always
```
Comment (jennyrosewood) [inline tutorials/cloudbuild-test-runner.md]: Change we to you in this sentence and throughout this doc

**Pull request merged**